core/commands/custom.py

```diff
@@ -68,22 +57,26 @@ def run_async(self,
             elif arg == "{PROMPT_ARG}":
                 args[idx] = custom_argument
 
-        self.window.status_message(start_msg)
-        if run_in_thread:
-            stdout = ''
-            cmd_thread = CustomCommandThread(self.git, *args, custom_environ=custom_environ)
-            cmd_thread.start()
-        else:
+        def program():
+            self.window.status_message(start_msg)
             stdout = self.git(*args, custom_environ=custom_environ)
-        self.window.status_message(complete_msg)
-
-        if output_to_panel:
-            util.log.panel(stdout)
-        if output_to_buffer:
-            view = self.window.new_file()
-            view.set_scratch(True)
-            view.run_command("gs_replace_view_text", {"text": stdout.replace("\r", "\n"), "nuke_cursors": True})
-            if syntax:
-                view.set_syntax_file(syntax)
+            self.window.status_message(complete_msg)
+
+            if output_to_panel:
+                util.log.panel(stdout)
+            if output_to_buffer:
+                view = self.window.new_file()
+                view.set_scratch(True)
+                view.run_command("gs_replace_view_text", {
+                    "text": stdout.replace("\r", "\n"),
+                    "nuke_cursors": True,
+                })
+                if syntax:
+                    view.set_syntax_file(syntax)
+
+            util.view.refresh_gitsavvy_interfaces(self.window)
 
-        util.view.refresh_gitsavvy(self.window.active_view())
+        if run_in_thread:
+            threading.Thread(target=program, daemon=True).start()
```

Comment:
Question: why don't we use the `savvy_executor` via [`enqueue_on_savvy` or `run_on_new_thread`](https://github.com/timbrel/GitSavvy/blob/38dd8c5c10bd3616d5f57f1dd4296601dee53c8b/core/runtime.py#L38-L45) instead of `threading.Thread` directly here?

Reply:
I wanted but (after reading the PR) `daemon=True` is a requirement here. (The example was: "It is good that Git GUI does *NOT* close automatically when Sublime exits."